data/query_data.py

```
@@ -0,0 +1,73 @@
+import os
+from langchain_google_genai import GoogleGenerativeAI
+from langchain.chains import SimpleSequentialChain
```

Ruff (0.8.2)

`langchain.chains.SimpleSequentialChain` imported but unused

Remove unused import: `langchain.chains.SimpleSequentialChain`

(F401)